features/order/making_an_order/making_an_order_as_a_customer.feature

```diff
@@ -0,0 +1,21 @@
+@making_orders
+Feature: Making an order
+    In order to know about new orders
+    As an Administrator
+    I want to be able to manage new orders made by customers
+
+    Background:
+        Given the store operates on a single channel in "France"
+        And the store has a product "PHP T-Shirt" priced at "$19.99"
+        And the store ships everywhere for free
+        And the store allows paying offline
+        And there is a user "john.doe@example.com"
+        And this user has this product in the cart
+        And I am logged in as an administrator
+
+    @ui
+    Scenario: Verifying that order has new state
+        Given this user bought this product
```

Comment:
I know that user and products are in backgroud, but this step look very strange. 
Maybe move steps about product and user from background to scenario?